2024-03-27 23:58:09,814 INFO    StreamThr :2172 [internal.py:wandb_internal():86] W&B internal server running at pid: 2172, started at: 2024-03-27 23:58:09.811000
2024-03-27 23:58:09,823 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status
2024-03-27 23:58:09,836 INFO    WriterThread:2172 [datastore.py:open_for_write():87] open: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240327_235809-ix617jmg\run-ix617jmg.wandb
2024-03-27 23:58:09,836 DEBUG   SenderThread:2172 [sender.py:send():378] send: header
2024-03-27 23:58:09,852 DEBUG   SenderThread:2172 [sender.py:send():378] send: run
2024-03-27 23:58:11,113 INFO    SenderThread:2172 [dir_watcher.py:__init__():211] watching files in: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240327_235809-ix617jmg\files
2024-03-27 23:58:11,113 INFO    SenderThread:2172 [sender.py:_start_run_threads():1099] run started: ix617jmg with start time 1711564089.820209
2024-03-27 23:58:11,179 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: check_version
2024-03-27 23:58:11,185 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: check_version
2024-03-27 23:58:12,202 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: run_start
2024-03-27 23:58:13,008 DEBUG   HandlerThread:2172 [system_info.py:__init__():26] System info init
2024-03-27 23:58:13,008 DEBUG   HandlerThread:2172 [system_info.py:__init__():41] System info init done
2024-03-27 23:58:13,008 INFO    HandlerThread:2172 [system_monitor.py:start():194] Starting system monitor
2024-03-27 23:58:13,008 INFO    SystemMonitor:2172 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-03-27 23:58:13,008 INFO    HandlerThread:2172 [system_monitor.py:probe():214] Collecting system info
2024-03-27 23:58:13,024 INFO    SystemMonitor:2172 [interfaces.py:start():190] Started cpu monitoring
2024-03-27 23:58:13,024 INFO    SystemMonitor:2172 [interfaces.py:start():190] Started disk monitoring
2024-03-27 23:58:13,024 INFO    SystemMonitor:2172 [interfaces.py:start():190] Started gpu monitoring
2024-03-27 23:58:13,024 INFO    SystemMonitor:2172 [interfaces.py:start():190] Started memory monitoring
2024-03-27 23:58:13,024 INFO    SystemMonitor:2172 [interfaces.py:start():190] Started network monitoring
2024-03-27 23:58:13,040 DEBUG   HandlerThread:2172 [system_info.py:probe():150] Probing system
2024-03-27 23:58:13,040 DEBUG   HandlerThread:2172 [gitlib.py:_init_repo():56] git repository is invalid
2024-03-27 23:58:13,040 DEBUG   HandlerThread:2172 [system_info.py:probe():198] Probing system done
2024-03-27 23:58:13,040 DEBUG   HandlerThread:2172 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22631-SP0', 'python': '3.9.13', 'heartbeatAt': '2024-03-27T18:28:13.040346', 'startedAt': '2024-03-27T18:28:09.772241', 'docker': None, 'cuda': None, 'args': ('8000',), 'state': 'running', 'program': 'C:\\Users\\chinm\\Desktop\\FL-FINAL\\CIFAR\\server.py', 'codePathLocal': 'server.py', 'codePath': 'server.py', 'host': 'MSI', 'username': 'chinm', 'executable': 'C:\\Users\\chinm\\anaconda3\\New folder\\python.exe', 'cpu_count': 6, 'cpu_count_logical': 12, 'cpu_freq': {'current': 2496.0, 'min': 0.0, 'max': 2496.0}, 'cpu_freq_per_core': [{'current': 2496.0, 'min': 0.0, 'max': 2496.0}], 'disk': {'/': {'total': 458.81444931030273, 'used': 430.0235481262207}}, 'gpu': 'NVIDIA GeForce RTX 3050 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3050 Laptop GPU', 'memory_total': 4294967296}], 'memory': {'total': 7.8259735107421875}}
2024-03-27 23:58:13,040 INFO    HandlerThread:2172 [system_monitor.py:probe():224] Finished collecting system info
2024-03-27 23:58:13,040 INFO    HandlerThread:2172 [system_monitor.py:probe():227] Publishing system info
2024-03-27 23:58:13,040 DEBUG   HandlerThread:2172 [system_info.py:_save_conda():207] Saving list of conda packages installed into the current environment
2024-03-27 23:58:13,055 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-27 23:58:13,144 INFO    Thread-16 :2172 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240327_235809-ix617jmg\files\conda-environment.yaml
2024-03-27 23:58:15,652 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:15,652 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:18,274 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:18,274 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:20,931 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:20,931 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:23,550 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:23,550 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:26,166 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:26,166 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:28,810 DEBUG   HandlerThread:2172 [system_info.py:_save_conda():219] Saving conda packages done
2024-03-27 23:58:28,810 INFO    HandlerThread:2172 [system_monitor.py:probe():229] Finished publishing system info
2024-03-27 23:58:28,810 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:28,810 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:28,822 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:58:28,822 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: keepalive
2024-03-27 23:58:28,822 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:58:28,822 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: keepalive
2024-03-27 23:58:28,822 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:58:28,822 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: keepalive
2024-03-27 23:58:28,822 DEBUG   SenderThread:2172 [sender.py:send():378] send: files
2024-03-27 23:58:28,822 INFO    SenderThread:2172 [sender.py:_save_file():1365] saving file wandb-metadata.json with policy now
2024-03-27 23:58:29,278 INFO    Thread-16 :2172 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240327_235809-ix617jmg\files\conda-environment.yaml
2024-03-27 23:58:29,278 INFO    Thread-16 :2172 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240327_235809-ix617jmg\files\wandb-metadata.json
2024-03-27 23:58:29,817 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: python_packages
2024-03-27 23:58:29,817 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: python_packages
2024-03-27 23:58:29,817 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: stop_status
2024-03-27 23:58:29,817 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-27 23:58:29,817 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: stop_status
2024-03-27 23:58:30,278 INFO    Thread-16 :2172 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240327_235809-ix617jmg\files\requirements.txt
2024-03-27 23:58:30,278 DEBUG   SenderThread:2172 [sender.py:send():378] send: telemetry
2024-03-27 23:58:30,483 INFO    wandb-upload_0:2172 [upload_job.py:push():131] Uploaded file C:\Users\chinm\AppData\Local\Temp\tmptsl9bvgmwandb\dm3xt7fi-wandb-metadata.json
2024-03-27 23:58:31,288 INFO    Thread-16 :2172 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240327_235809-ix617jmg\files\output.log
2024-03-27 23:58:31,303 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:58:31,445 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:31,445 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:32,290 INFO    Thread-16 :2172 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240327_235809-ix617jmg\files\output.log
2024-03-27 23:58:34,074 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:34,074 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:36,349 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:58:36,702 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:36,702 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:39,310 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:39,310 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:41,380 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:58:41,936 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:41,936 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:42,359 INFO    Thread-16 :2172 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240327_235809-ix617jmg\files\config.yaml
2024-03-27 23:58:44,377 INFO    Thread-16 :2172 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240327_235809-ix617jmg\files\output.log
2024-03-27 23:58:44,566 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:44,566 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:44,828 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: stop_status
2024-03-27 23:58:44,828 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-27 23:58:44,828 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: stop_status
2024-03-27 23:58:47,202 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:47,202 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:47,242 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:58:49,841 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:49,841 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:52,501 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:58:52,565 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-27 23:58:55,217 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:55,217 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:57,531 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:58:57,903 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:57,910 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:58:59,833 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-27 23:58:59,833 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: stop_status
2024-03-27 23:58:59,833 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: stop_status
2024-03-27 23:59:00,554 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:00,554 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:03,203 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:03,203 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:03,301 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:59:05,850 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:05,850 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:07,032 INFO    Thread-16 :2172 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240327_235809-ix617jmg\files\output.log
2024-03-27 23:59:08,402 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:59:08,504 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:08,511 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:11,153 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:11,154 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:13,042 DEBUG   SystemMonitor:2172 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-03-27 23:59:13,049 DEBUG   SenderThread:2172 [sender.py:send():378] send: stats
2024-03-27 23:59:13,823 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:13,824 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:14,062 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:59:14,834 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: stop_status
2024-03-27 23:59:14,834 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: stop_status
2024-03-27 23:59:14,843 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-27 23:59:16,492 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:16,492 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:19,165 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:19,165 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:19,317 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:59:21,820 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:21,820 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:24,388 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:59:24,509 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:24,509 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:27,167 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:27,170 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:29,452 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:59:29,836 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: stop_status
2024-03-27 23:59:29,837 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: stop_status
2024-03-27 23:59:29,847 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:29,849 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:29,851 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-27 23:59:32,607 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-27 23:59:35,277 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:35,277 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:35,337 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:59:37,980 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:37,980 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:40,379 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:59:40,697 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-27 23:59:43,068 DEBUG   SenderThread:2172 [sender.py:send():378] send: stats
2024-03-27 23:59:43,411 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:43,417 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:44,842 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: stop_status
2024-03-27 23:59:44,842 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: stop_status
2024-03-27 23:59:44,862 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-27 23:59:45,548 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:59:46,140 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:46,140 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:48,807 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:48,807 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:50,662 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:59:51,506 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:51,508 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:54,258 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:54,258 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:55,812 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-27 23:59:56,971 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:56,971 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-27 23:59:59,751 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-27 23:59:59,861 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: stop_status
2024-03-27 23:59:59,863 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: stop_status
2024-03-27 23:59:59,874 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:00:01,714 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:00:02,429 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:02,429 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:05,195 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:00:06,763 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:00:07,921 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:07,929 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:10,650 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:10,650 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:14,649 DEBUG   SenderThread:2172 [sender.py:send():378] send: stats
2024-03-28 00:00:14,649 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:00:14,856 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:00:14,867 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:00:14,898 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:00:14,899 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:00:18,603 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:00:20,422 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:00:21,323 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:21,323 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:24,093 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:00:25,187 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:00:26,847 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:00:29,621 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:29,630 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:29,882 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:00:29,899 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:00:29,900 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:00:30,300 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:00:32,477 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:32,477 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:35,181 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:35,182 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:35,334 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:00:37,858 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:37,863 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:40,385 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:00:40,594 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:40,594 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:43,163 DEBUG   SenderThread:2172 [sender.py:send():378] send: stats
2024-03-28 00:00:43,352 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:00:44,902 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:00:44,930 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:00:44,930 DEBUG   SenderThread:2172 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:00:45,633 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:00:46,152 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:00:48,997 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:00:50,842 DEBUG   HandlerThread:2172 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:00:51,707 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:00:51,715 ERROR   gpu       :2172 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
